### ISSUE TITLE
Dialog and Panel accessibility issues

### ISSUE THREAD

**Issue by jasonmac19** [microsoft/powercat-creator-kit#25]
The dialog and panel components seem to suffer from the issues highlighted in the link below, specifically under the 'Dialogs and overlays' section. Is there any thought as to making these components accessible? Perhaps they could be PCFs?

https://docs.microsoft.com/en-us/power-apps/maker/canvas-apps/accessible-apps-limitations

The company I work for has a legal requirement to conform to WACG 2.1 so we would be unable to make use of these components.

**Comment by denisem-msft** [comment]
If dialogs are necessary for your design, you can create custom pages for dialogs and panels in a model driven app with custom pages: https://learn.microsoft.com/en-us/power-apps/developer/model-driven-apps/clientapi/navigate-to-custom-page-examples

We are evaluating whether we can implement panel and dialog in code components based on resourcing. We will keep this item open until we have more clarity around resources for this.

**Comment by mikefactorial** [comment]
@jasonmac19 This has been fixed in the latest release.

**Comment by denise-msft** [comment]
Apologies, this was not fixed in the latest release - please look into the model driven app capabilities suggested to address accessibility.

Since we don't have a good answer in the current capabilities for Power Apps, the recommendation to meet accessibility requirements is to put any information that should be in a dialog or panel on a new screen.
